scintilla/src/Editor.cxx

```diff
@@ -4675,6 +4676,7 @@ void Editor::ButtonDownWithModifiers(Point pt, unsigned int curTime, int modifie
 	lastClick = pt;
 	lastXChosen = static_cast<int>(pt.x) + xOffset;
 	ShowCaretAtCurrentPosition();
+	if (inSelMargin) SetHoverIndicatorPosition(INVALID_POSITION);

```

Comment:
One more stuff, in order to make future Scintilla updating easier, could you add the following comment:
// Fix the following issue:
// https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8647
// It could be fixed in Scintilla release superior to v4.4.4 - to verify.